x-pack/plugins/security_solution/common/endpoint/service/authz/authz.ts

```diff
@@ -70,13 +62,26 @@ export const calculateEndpointAuthz = (
   fleetAuthz: FleetAuthz,
   userRoles: MaybeImmutable<string[]>,
   isEndpointRbacEnabled: boolean = false,
-  permissions: Partial<EndpointPermissions> = defaultEndpointPermissions(),
   hasHostIsolationExceptionsItems: boolean = false
 ): EndpointAuthz => {
   const isPlatinumPlusLicense = licenseService.isPlatinumPlus();
   const isEnterpriseLicense = licenseService.isEnterprise();
   const hasEndpointManagementAccess = userRoles.includes('superuser');
-  const { canWriteSecuritySolution = false, canReadSecuritySolution = false } = permissions;
+
+  const canWriteSecuritySolution = hasKibanaPrivilege(
+    fleetAuthz,
+    isEndpointRbacEnabled,
+    hasEndpointManagementAccess,
+    'writeSecuritySolution'
```

Comment:
Does this and the one below needs the `isEndpointRbacEnabled` check? This privilege already existed before, right?

Reply:
you're right, will update.